src/components/Auth/api.ts

```diff
@@ -54,8 +55,34 @@ export default class AuthApi extends RestApi {
     super();
   }
 
+  async nukeUsersForAllTenants() {
+    const deletePromises = [];
+
+    // clear users from default tenant
+    deletePromises.push(this.deleteRequest(`${this.baseEmulatorUrl}/accounts`));
+
+    // clear users from all other tenants
+    const tenants = await this.fetchTenants();
+    const tenantDeletes = tenants.map(({ tenantId }) => {
+      return this.deleteRequest(
+        `${this.baseEmulatorUrl}/tenants/${tenantId}/accounts`
+      );
+    });
+    deletePromises.concat(tenantDeletes);
+
+    await Promise.all(deletePromises);
+    return [];
```

Comment:
nit: don't return this and just change function signature to return `Promise<void>`